### PR TITLE
Tcintra working branch

### PR DESCRIPTION
**Pull request by Tcintra**
Updated front page with coursework, skills, and media buttons. Added resume, pfp, media links, and a resume download button to the resume page. Tweaked side navbar so it only starts below the header, but this required me to specify the number of pixels on the "top" margin which I think is bad style.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,37 +1,94 @@
...
+					</form>
+					<form action="https://www.linkedin.com/in/thomascintra/" target="_blank">
+						<button type="submit" class="media-btns"><i class="fab fa-linkedin fa-3x"></i></button>
+					</form>
+					<form action="https://instagram.com/tncintra" target="_blank">
+						<button type="submit" class="media-btns"><i class="fab fa-instagram-square fa-3x"></i></button>
+					</form>
+				</div>
+            </div>
+		</div>
```
Comment (serzhk) [inline portfolio/src/main/webapp/index.html]: Is it possible to apply some text formatting? It seems some tags are outlined (you could treat open, closing tags as curly braces in any programming language)

Comment (Tcintra) [reply portfolio/src/main/webapp/index.html]: By text formatting do you mean fixing indentation and such? I tried putting it all through an html beautifier but I wasn't sure what Google's style guidelines for html were.

Comment (serzhk) [reply portfolio/src/main/webapp/index.html]: Yes, indentation. There is no strict guidelines, but overall it's better to keep indentation correct. I think there should be multiple options in the text editors to keep formatting correct. Also there are multiple online resources, where you put your code and it automatically format it: https://www.freeformatter.com/html-formatter.html

Comment (Tcintra) [reply portfolio/src/main/webapp/index.html]: Okay sounds good! I used http://beautifytools.com/html-beautifier.php but I don't think this website helped much, I'll use freeformatter.

Comment (Tcintra) [reply portfolio/src/main/webapp/index.html]: I pushed a new set of commits with some changes to home, new pages and hopefully fixed formatting issues. The command shift+ctrl+f (for me its shift+ctrl+i) wasn't doing much so I used freeformatter.

**Review by serzhk — APPROVED** [review]

**Pull request merged**